index.js

```diff
@@ -0,0 +1,132 @@
+// dependencies
+const inquirer = require("inquirer");
+const fs = require("fs");
+const createHTML = require("./src/html-creator.js");
```

Comment:
Remember to separate your internal and external imports by leaving a blank line in between